src/engraving/libmscore/layoutbreak.cpp

```diff
@@ -134,8 +134,10 @@ void LayoutBreak::draw(mu::draw::Painter* painter) const
         return;
     }
 
-    Pen pen;
-    pen.setColor(selected() ? engravingConfiguration()->selectionColor() : engravingConfiguration()->formattingMarksColor());
+    Pen pen(selected() ? engravingConfiguration()->selectionColor() : engravingConfiguration()->formattingMarksColor());
+    if (score()->isPaletteScore()) {
+        pen.setColor(engravingConfiguration()->themeColor(ui::ThemeStyleKey::FONT_PRIMARY_COLOR));
```

Comment:
better add method like `fontPrimaryColor` to `engravingConfiguration` (without arguments)

Reply:
Okay, I did think of that. Thank you!